src/lib/parsers/Args.ts

```diff
@@ -14,7 +14,7 @@ import type {
 	User,
 	VoiceChannel
 } from 'discord.js';
-import type * as Lexure from 'lexure';
+import type * as Lexure from '@sapphire/lexure';
```

Comment:
You not longer need to do this. The reason why I did this was because `lexure` exported `Args`, which conflicted with this class's name. Since I renamed it to `ArgumentStream`, we can now import everything top level.